data/json/items/comestibles/wheat.json

```diff
@@ -506,7 +731,12 @@
     "id": "sponge_cake",
     "type": "COMESTIBLE",
     "comestible_type": "FOOD",
-    "name": "sponge cake",
+    "name": { "str": "sponge cake" },
+    "conditional_names": [
+      { "type": "COMPONENT_ID", "condition": "honey", "name": { "str": "honey sponge cake" } },
+      { "type": "COMPONENT_ID", "condition": "owlbear", "name": { "str": "owlbear sponge cake" } },
```

Comment:
Does this not throw an error when Magiclysm isn't loaded?

Reply:
And if it doesn't, it possibly should. This stuff shouldn't be specified in the base game, it should be specified in mods.

Reply:
No. It doesn't cause any errors. The game isn't actually checking for ids, it's parsing the ids of ingredients of an item for a matching string so if I had it parsing for "foo" it would find "foofas" or "fafoos" Even if "foo" doesn't exist as an id, nothing happens. This just lets it be compatible with modded content without making another entry for literally every egg recipe. Seems intentional.

Reply:
Put it another way: I could dedicate several extra lines of json and files for every comestible that might included a conditional name for a modded ingredient or I can have 1 line in a main file with zero conflicts the player won't see or be affected by in any way. 

I cause more chaos with this PR by altering the recipe of ice cream than by adding a dead-end conditional name.